specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/examples/DataSourcesListByWorkspace.json

```diff
@@ -0,0 +1,3019 @@
+   },
+   "responses":{
+      "200":{
+         "body":{
+            "value":[
+               {
+                  "kind":"WindowsEvent",
+                  "properties":{
+                     "eventLogName":"windowsEvent14",
+                     "eventTypes":[
+                        {
+                           "eventType":"Error"
+                        }
+                     ]
+                  },
+                  "id":"/subscriptions/00000000-0000-0000-0000-000000000005/resourceGroups/OIAutoRest7887/providers/Microsoft.OperationalInsights/workspaces/AzTest218/datasources/AzTestDSWE1011",
+                  "etag":"W/\"datetime'2017-10-02T23%3A20%3A08.5629323Z'\"",
+                  "name":"AzTestDSWE1011",
+                  "type":"Microsoft.OperationalInsights/workspaces/datasources"
+               },
+               {
+                  "kind":"WindowsEvent",
+                  "properties":{
+                     "eventLogName":"windowsEvent64",
+                     "eventTypes":[
+                        {
+                           "eventType":"Error"
+                        }
+                     ]
+                  },
+                  "id":"/subscriptions/00000000-0000-0000-0000-000000000005/resourceGroups/OIAutoRest7887/providers/Microsoft.OperationalInsights/workspaces/AzTest218/datasources/AzTestDSWE1013",
+                  "etag":"W/\"datetime'2017-10-02T23%3A20%3A22.2533211Z'\"",
+                  "name":"AzTestDSWE1013",
+                  "type":"Microsoft.OperationalInsights/workspaces/datasources"
+               },
+               {
+                  "kind":"WindowsEvent",
+                  "properties":{
+                     "eventLogName":"windowsEvent202",
+                     "eventTypes":[
+                        {
+                           "eventType":"Error"
+                        }
+                     ]
+                  },
+                  "id":"/subscriptions/00000000-0000-0000-0000-000000000005/resourceGroups/OIAutoRest7887/providers/Microsoft.OperationalInsights/workspaces/AzTest218/datasources/AzTestDSWE1020",
+                  "etag":"W/\"datetime'2017-10-02T23%3A21%3A04.4645698Z'\"",
+                  "name":"AzTestDSWE1020",
+                  "type":"Microsoft.OperationalInsights/workspaces/datasources"
+               },
+               {
+                  "kind":"WindowsEvent",
+                  "properties":{
+                     "eventLogName":"windowsEvent231",
+                     "eventTypes":[
+                        {
+                           "eventType":"Error"
+                        }
+                     ]
+                  },
+                  "id":"/subscriptions/00000000-0000-0000-0000-000000000005/resourceGroups/OIAutoRest7887/providers/Microsoft.OperationalInsights/workspaces/AzTest218/datasources/AzTestDSWE1074",
+                  "etag":"W/\"datetime'2017-10-02T23%3A21%3A12.5871672Z'\"",
```

Comment:
One more comment, I'm still seeing some mixed casing between `eTag` and `etag`. Let's resolve these all to be consistent with whichever way it is on the server.

Reply:
I changed them all to eTag and updated the examples. Please see next iteration.